pouta_blueprints/views/stats.py

```diff
@@ -0,0 +1,71 @@
+from flask.ext.restful import marshal_with, fields
+from flask import Blueprint as FlaskBlueprint
+
+import logging
+
+from pouta_blueprints.models import Blueprint, Instance
+from pouta_blueprints.server import restful
+from pouta_blueprints.views.commons import auth
+from pouta_blueprints.utils import requires_admin, memoize
+
+from collections import defaultdict
+
+
+stats = FlaskBlueprint('stats', __name__)
+
+
+def query_blueprint(blueprint_id):
+    return Blueprint.query.filter_by(id=blueprint_id).first()
+
+
+blueprint_result_fields = {
```

Comment:
there should be blueprint_stats_fields with object like {'id': fields.String, 'name': fields.String, 'users': fields.Integer, 'launched_instances': fields.Integer, 'running_instances': fields.Integer} and something like stats_fields with {'blueprints': fields.Nested(blueprint_stats_fields), 'overall_running_instances': fields.Integer}